node/src/accountant/scanners/scanners_utils.rs

```diff
@@ -843,4 +871,62 @@ mod tests {
             "Got 0 properly sent payables of an unknown number of attempts"
         )
     }
+
+    #[test]
+    fn requires_payments_retry_says_yes() {
+        let cases = vec![
+            PendingPayableScanReport {
+                still_pending: vec![PendingPayableId::new(12, make_tx_hash(456))],
+                failures: vec![],
+                confirmed: vec![],
+            },
+            PendingPayableScanReport {
+                still_pending: vec![],
+                failures: vec![PendingPayableId::new(456, make_tx_hash(1234))],
+                confirmed: vec![],
+            },
+            PendingPayableScanReport {
+                still_pending: vec![PendingPayableId::new(12, make_tx_hash(456))],
+                failures: vec![PendingPayableId::new(456, make_tx_hash(1234))],
+                confirmed: vec![],
+            },
```

Comment:
You's like to include the 2 remaining cases too:

1. All empty - You might want to just leave a comment mentioning, that'll never happen.
2. Confirmed is non-empty and the other two are empty: You'd like to consider asserting on this one.